docs/getting_started/limitations.md

```diff
@@ -1,5 +1,13 @@
 While the Government PaaS is built using Cloud Foundry technology, we don't support all Cloud Foundry features. This section explains some Cloud Foundry features that are not enabled, as well as some limitations of the beta phase.
 
+## IP whitelisting
```

Comment:
Is IP whitelisting well understood enough terminology? Could we make this clearer by making the title something like "you can only manage apps from certain locations / networks"

I wonder if "restrictions" is a slightly plainer English word that could be a drop in for whitelist in most of this text